tests/ui/mock/performance_testData.json

```diff
@@ -0,0 +1,186 @@
+    "projectId": 1,
+    "id": "mozilla-central perf_reftest_singletons opt e10s stylo",
+    "data": [
+      {
+        "x": "2020-02-26T00:43:32.000Z",
+        "y": 73.77654318687213,
+        "z": "#464876",
+        "_z": ["circle", "fill"],
+        "revision": "f3da8ae9d1a3e74cd273746da51a035ddc572bee",
+        "signature_id": 1646973,
+        "pushId": 649011,
+        "jobId": 290442097,
+        "dataPointId": 1058414177,
+        "application": null
+      },
+      {
+        "x": "2020-02-26T12:27:57.000Z",
+        "y": 73.79306021156461,
+        "z": "#464876",
+        "_z": ["circle", "fill"],
+        "revision": "7f41334e10443f4f1c7426e86fb0cb7adfdf4d62",
+        "signature_id": 1646973,
+        "pushId": 649409,
+        "jobId": 290533148,
+        "dataPointId": 1059078029,
+        "application": null
+      },
+      {
+        "x": "2020-02-26T19:25:51.000Z",
+        "y": 73.92300809110661,
+        "z": "#464876",
+        "_z": ["circle", "fill"],
+        "revision": "94c8f28a15e8a051aeaad4722deb3fcc8125dab2",
+        "signature_id": 1646973,
+        "pushId": 649765,
+        "jobId": 290604490,
+        "dataPointId": 1059446203,
+        "application": null
+      },
+      {
+        "x": "2020-02-27T00:39:03.000Z",
+        "y": 73.5636612585284,
+        "z": "#464876",
+        "_z": ["circle", "fill"],
+        "revision": "862da1751d9fb10d1daa20940ffa722c888078b1",
+        "signature_id": 1646973,
+        "pushId": 650087,
+        "jobId": 290656251,
+        "dataPointId": 1059785529,
+        "application": null
+      },
+      {
+        "x": "2020-02-27T06:39:37.000Z",
+        "y": 73.95004892162557,
+        "z": "#464876",
+        "_z": ["circle", "fill"],
+        "revision": "7a5cb26a2d518e9cfaf512ba9a06239b573d7f0e",
+        "signature_id": 1646973,
+        "pushId": 650358,
+        "jobId": 290709821,
+        "dataPointId": 1060129998,
+        "application": null
+      },
+      {
+        "x": "2020-02-27T12:49:56.000Z",
+        "y": 73.62039234066732,
+        "z": "#464876",
+        "_z": ["circle", "fill"],
+        "revision": "9e8d5431c4121a4bd70d440c98b50444aee60dd9",
+        "signature_id": 1646973,
+        "pushId": 650506,
+        "jobId": 290734931,
+        "dataPointId": 1060279206,
+        "application": null
+      }
+    ],
+    "application": null,
+    "measurementUnit": "",
+    "lowerIsBetter": true,
+    "resultSetData": [649011, 649409, 649765, 650087, 650358, 650506],
+    "parentSignature": null
+  },
+  {
+    "color": ["cerulean", "#16BCDE"],
+    "symbol": ["circle", "outline"],
+    "visible": true,
+    "name": "tart_flex opt e10s stylo",
+    "signature_id": 1647462,
+    "signatureHash": "664c274bfcf996b14aa7771b6f2779531c309868",
+    "framework_id": 1,
+    "platform": "linux64",
+    "repository_name": "mozilla-central",
+    "projectId": 1,
+    "id": "mozilla-central tart_flex opt e10s stylo",
+    "data": [
```

Comment:
Instead of adding all of this data, I think you can use the `createGraphData` function with existing test data. Look in the tests/ui/perfherder/graphs_view_test file.